src/uv-common.c

```diff
@@ -223,7 +226,7 @@ int uv_ip6_addr(const char* ip, int port, struct sockaddr_in6* addr) {
   memset(addr, 0, sizeof(*addr));
   addr->sin6_family = AF_INET6;
   addr->sin6_port = htons(port);
-#ifdef SIN6_LEN
+#if defined(SIN6_LEN) || defined(__APPLE__) || defined(__FreeBSD__) || defined(__DragonFly__)
```

Comment:
Is the `__FreeBSD__`  check needed? When I added the `SIN6_LEN` ifdef, it worked on `FreeBSD`.

Reply:
`SIN6_LEN` also worked on my mac so I guess `__FreeBSD__ ` and `__DragonFly__ ` are also not necessary?

And I'm looking for a `SIN_LEN` for ipv4 buf couldn't find one.

Reply:
Maybe using `SIN_LEN6` also for `ipv4`?